Extensions/DialogueTree/dialoguetools.js

```diff
@@ -686,7 +711,9 @@ gdjs.dialogueTree.loadState = function(inputVariable) {
       gdjs.evtTools.network.variableStructureToJSON(inputVariable)
     );
     gdjs.dialogueTree.runner.visited = loadedState.visited;
-    gdjs.dialogueTree.runner.variables.data = loadedState.variables;
+    Object.entries(gdjs.dialogueTree.runner.variables).forEach(function([key, value]) {
```

Comment:
This array destructuring is not ES5 compatible.
https://github.com/4ian/GDevelop/blob/master/newIDE/docs/Supported-JavaScript-features-and-coding-style.md#for-the-game-engine-gdjs-and-extensions

In the game engine, we can't use arrow functions, let/const or destructuring.

Reply:
ah good catch. I've gotten so used to deconstructing now it's sometimes hard to stop myself

Reply:
fixed now